tests/logging_int/main.c

```diff
@@ -6,10 +6,9 @@
 /*not much in the way of testing, but "should not crash" */
 int main(void)
 {
-
-    char really_big_string[4096]; /*4096 (the size of this array) needs top be bigger than LOG_SIZE_REGULAR from etwlogger_driver.c*/
+    char really_big_string[LOG_SIZE_REGULAR/2]; /*if the size of this array is greater than LOG_SIZE_REGULAR than it will not get printed as all memory is statically allocated and cannot grow*/
```

Comment:
>*if the size of this array is greater than LOG_SIZE_REGULAR than it will not get printed as all memory is statically allocated and cannot grow*/ [](start = 49, length = 144)

Shouldn't we have a test for that?

Reply:
We absolutely should. Added a test where the big string canot be printed because it exceeds the size of the buffer.

Also added LogWarning. LogVerbose and LogCritical to this file.

---
In reply to: [453754805](https://github.com/Azure/azure-c-logging/pull/11#discussion_r453754805) [](ancestors = 453754805)